index.js

```diff
@@ -0,0 +1,311 @@
+const newTask = document.getElementById('new-task');
+const addTaskButton = document.getElementById('add-task');
+const editTaskButton = document.getElementById('edit-task');
+const title = document.getElementById('inputTitle');
+const text = document.getElementById('inputText');
+const taskField = document.getElementById('currentTasks');

```

Comment:
Could you tell why you variables names so different from class names ? Logically the should mean almost the same 
`const currentTasks = ..... ('currentTasks')` and etc. Also give semantically clear and correct names. Please, read in internet what does semantic means if it's not clear.

Reply:
thanks!
fixed it
```suggestion
const currentTasks = document.getElementById('currentTasks');
```